sys/parsoid.js

```diff
@@ -227,15 +216,7 @@ class ParsoidService {
     }
 
     getFallbackBucketURI(rp, format, tid) {
-        // TODO: We're adding the -ng only to avoid table name clashing in dev cluster. Remove!
-        const path = [rp.domain, 'sys', 'key_rev_value', `parsoid.stash.${format}-ng`, rp.title];
-        if (rp.revision) {
-            path.push(rp.revision);
-            if (tid) {
-                path.push(tid);
-            }
-        }
-        return new URI(path);
+        return this.getStashBucketURI(rp, format, tid);
```

Comment:
Hm, the new tables have been created with the `-ng` suffix, so we need to use it here.